### ISSUE TITLE
Expand button should have discernible Text

### ISSUE THREAD

**Issue by adagar** [storybookjs/storybook#13797]
**Is your feature request related to a problem? Please describe**
The expand buttons in the sidebar are currently flagged as failing the Axe test for discernible text. This is a problem because a user who is using a11y tools to navigate may not have the purpose of the button be apparent to them.

**Describe the solution you'd like**
To maintain current visuals, an `aria-label` attribute could be added. A test with this change passes Storybook's and axe's a11y checks
![Screen Shot 2021-02-02 at 5 01 57 PM](https://user-images.githubusercontent.com/3698858/106668575-6d599380-6578-11eb-80ea-ea2dcb57d651.png)
![Screen Shot 2021-02-02 at 5 01 49 PM](https://user-images.githubusercontent.com/3698858/106668577-6df22a00-6578-11eb-9d63-8684a84a393e.png)

**Describe alternatives you've considered**
A solution that would benefit both sighted and non-sighted users would be to have visible text associated with the expand button. This would unfortunately change the current visual appearance of the tool though.

**Are you able to assist to bring the feature to reality?**
Yes, should be a pretty straightforward change

**Additional context**

**Comment by shilman** [comment]
Ta-da!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.2.0-rc.10 containing PR #14338 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```

Closing this issue. Please re-open if you think there's still more to do.
